content-elements/form/checkbox/prototype/checkbox.js

```diff
@@ -8,10 +8,11 @@ Alpine.data('checkbox', () => ({
   },
 
   validateInput() {
-    if (!this.inputEl.value && this.inputEl.hasAttribute('required')) {
-      this.requiredErrorElement.style.display = "block";
-    } else {
+    if (this.$el.checked || !this.$el.hasAttribute('required')) {
```

Comment:
Ich weiß, dass dieses if-else auch woanders im Code ist, aber ich finde inline schöner:
`this.requiredErrorElement.style.display = (this.$el.checked || !this.$el.hasAttribute('required')) ? 'none' : 'block'`